libtsuba/include/tsuba/RDG.h

```diff
@@ -74,6 +74,11 @@ class KATANA_EXPORT RDG {
 
   katana::Result<void> UnbindTopologyFileStorage();
 
+  /// Inform this RDG that it's topology is in storage at this location
```

Comment:
"its", "graph"

Reply:
Is this register or really just updating the topology altogether?

E.g., is this any different than how we add and remove properties?

Reply:
It is a little awkward. All of the existing operations for properties/topology say "here's data to be your new property/topology, write it out when we write out the next version". In this case I needed something that says "I've already written out the topology, here's where I stored it, update your metadata to reflect that"

Reply:
Maybe if we get more of these use-cases, one way to organize them is as "updating the metadata" versus "updating the data."

I think this will be more specific than "register". I don't know how important it is now though.

Reply:
I renamed the function `SetTopologyFilename` which I think at least limits the scope of what users should expect from this function.